frontend/app/services/personal-information-service.server.ts

```diff
@@ -68,37 +68,38 @@ const personalInformationApiResponseSchema = z.object({
 
 type PersonalInformationApiResponse = z.infer<typeof personalInformationApiResponseSchema>;
 
-const personalInfoSchema = z.object({
+const personalInfoDtoSchema = z.object({
   clientInfoList: z
     .object({
       id: z.string().optional(),
       category: z.string().optional(),
     })
     .array()
     .optional(),
-  personsNameList: z
+  lastName: z.string().optional(),
+  firstName: z.string().optional(),
+  homeAddress: z
     .object({
-      lastName: z.string().optional(),
-      firstName: z.string().optional(),
+      categoryCode: z.string(),
+      streetName: z.string(),
+      cityName: z.string(),
+      provinceName: z.string(),
```

Comment:
In the case of an international address:

```suggestion
      provinceName: z.string().optional(),
```

(same comment for `mailingAddress.provinceName`)